wp-includes/integrations/class-wp-service-worker-scripts-integration.php

```diff
@@ -12,6 +12,14 @@
  */
 class WP_Service_Worker_Scripts_Integration extends WP_Service_Worker_Base_Integration {
 
+	/**
+	 * Scope this integration applies to.
+	 *
+	 * @since 0.2
+	 * @var int
+	 */
+	protected $scope = WP_Service_Workers::SCOPE_ALL;
```

Comment:
This can be removed, as it is no longer necessary after #72. The scripts and styles integrations are now already available in both scopes.